src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/collector-configuration-examples/opentelemetry-collector-hcp-consul.mdx

```diff
@@ -0,0 +1,172 @@
+freshnessValidatedDate: never
+---
+
+You can collect metrics about your HCP-managed Consul deployment with the OpenTelemetry Collector. The collector is a component of OpenTelemetry that collects, processes, and exports telemetry data to New Relic (or any observability backend).
+
+This example works by running a prometheus receiver configuration inside the OpenTelemetry collector, which scrapes [HCP Consul's service metrics API](https://developer.hashicorp.com/hcp/docs/consul/monitor/metrics) and exports that data to [New Relic's OpenTelemetry API endpoint](https://docs.newrelic.com/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/opentelemetry-set-up-your-app/#note-endpoints).
+
+HCP Consul emits two forms of observability metrics, [Service metrics](https://developer.hashicorp.com/hcp/docs/consul/monitor/consul-central/observability#server-metrics) and [Envoy proxy metrics](https://developer.hashicorp.com/hcp/docs/consul/monitor/consul-central/observability#envoy-proxy-metrics). The example collector used in this guide will get you started with HCP Consul server metrics. For Envoy metrics, you can reference the notes below on Envoy metrics or check the [HCP Documentation](https://developer.hashicorp.com/hcp/docs/consul/monitor/consul-central/observability/telemetry-collector#forward-metrics-to-another-collector).
+
+Complete the steps below to collect metrics from HCP consul and export them to New Relic.
+
+<Steps>
+    <Step>
+        ## Make sure you're set up
+
+      Before you start, you need to have the <InlinePopover type="licenseKey" /> for the account you want to report data to. You should also verify that:
+
+      * You have a docker daemon running
+      * You have [Docker Compose](https://docs.docker.com/compose/) installed
+      * You have your [HCP Consul ACL admin token](https://developer.hashicorp.com/hcp/docs/consul/hcp-managed/access#generate-admin-token) available 
+
+    </Step>
+
+    <Step>
+        ## Download or clone the example repos
+
+        Download [New Relic's OpenTelemetry Examples repo](https://github.com/newrelic/newrelic-opentelemetry-examples) as this setup uses its example collector configuration. Once installed, open the [HCP Consul example](https://github.com/newrelic/newrelic-opentelemetry-examples/tree/main/other-examples/collector/hcp-consul) directory. For more information, you can check the `README` there as well.
+
+    </Step>
+
+    <Step>
+      ##  Set environment variables and run the collector
+        * Set the HCP Access Token and Access URL values in the `.env` file
+        * Set the New Relic API key variable to your New Relic license ingest key in the `.env` file
+
+```bash
+  # Open the HCP Consul example directory
+  cd newrelic-opentelemetry-examples/other-examples/collector/hcp-consul
+
+  # Set environment variables.
+
+  # run the collector in docker
+  docker compose up
```

Comment:
@jcountsNR  Can you add to this bash code block an example of the environment variables (mentioned in the bullet points above)?

Reply:
The environment variables are suggested to be set using the `.env` file in this case, which has comments and links to docs. It wouldn't be set from a bash command. It might make sense to just remove the comment from the command since it's specified above.